tests/integration/test_utils/data_source_config/databricks.py

```diff
@@ -68,21 +68,16 @@ def inferrable_types_lookup(self) -> dict[type, TypeEngine]:
     def _databrics_connection_config(self) -> DatabricksConnectionConfig:
         return DatabricksConnectionConfig()  # type: ignore[call-arg]  # retrieves env vars
 
+    @cached_property
```

Comment:
Does this being a cached property have significant performance implications? Is there any risk around caching here?

Reply:
I originally thought we would want to use the same asset if we access it multiple times. I'm not concerned about performance here at all TBH. I'm not totally sure we need it to be consistent across calls though. Maybe we end up making it just a `make_asset` method. I want to kick the can on that change to see what our needs are, but happy to do it now if you have opinions.